assets/yarn.lock

```diff
@@ -0,0 +1,146 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove this file, we're not using yarn on this project. `npm install` only please :+1:

Reply:
Done